hid-uclogic.c

```diff
@@ -544,6 +553,7 @@ enum uclogic_ph_id {
 /* Report descriptor template placeholder */
 #define UCLOGIC_PH(_ID) UCLOGIC_PH_HEAD, UCLOGIC_PH_ID_##_ID
 #define UCLOGIC_PEN_REPORT_ID	0x07
+#define UCLOGIC_HIRES_PEN_REPORT_ID	0x08
```

Comment:
I like the "hires" designation - the resolution bump seems to be exactly the reason they changed the string descriptor ID and format.